src/RewireState.js

```diff
@@ -12,13 +12,21 @@
  ACTION OF CONTRACT, NEGLIGENCE OR OTHER TORTIOUS ACTION, ARISING OUT OF
  OR IN CONNECTION WITH THE USE OR PERFORMANCE OF THIS SOFTWARE.*/
 
-import { universalAccesorsTemplate, enrichExportTemplate, filterWildcardImportTemplate } from './Templates.js';
+import Templates from './Templates.js';
 import { wasProcessed, noRewire } from './RewireHelpers.js';
-import * as t from 'babel-types';
+var t;
+var universalAccesorsTemplate;
```

Comment:
Why not use scope these variables to the `RewireState` class with `this.<variable>`, similar to `this.isES6Module` and the others? Also we should probably prefer `const` or `let` to `var`.

Reply:
I ran into issues with `this.t`, because of a few places it is referenced from unbound functions. Also it was a larger change. This way we just inject the lib at the top level when we get it from babel invoking us.